parse_and_import_medicover.py

```
@@ -74,12 +75,24 @@ def main(
     """
 
     db_creds = utils.parse_json(config_file)
     session, meta = db.connect_to_db(db_creds)
     inca_table = meta.tables["testdirectory.inca"]
 
+    # TODO fix this
+    if mapping_refAlt:
+        mapping_refAlt_data = utils.parse_tsv(
+            mapping_refAlt, "raw_ref", "raw_alt", "corrected_ref", "corrected_alt"
+        )
+        mapping_refAlt_dict = {
```

Ruff (0.15.2)

[error] Local variable `mapping_refAlt_dict` is assigned to but never used

Remove assignment to unused variable `mapping_refAlt_dict`

(F841)